edk2toolext/invocables/edk2_ci_build.py

```diff
@@ -26,13 +26,11 @@ class CiBuildSettingsManager(MultiPkgAwareSettingsInterface):
     ''' Platform settings will be accessed through this implementation. '''
 
     def GetActiveScopes(self):
-        ''' get scope '''
+        ''' return tuple containing scopes that should be active for this process '''
```

Comment:
maybe not tuple but iterable

Reply:
we add to it and we add a tuple so i think it must be a tuple